documentation/versioned_docs/version-1.5.3/commands/downloading-configuration.md

```diff
@@ -20,11 +20,11 @@ $ monaco download --environments=my-environment.yaml
 
 ### Options
 
-Instead of downloading all the configurations for all the API's you can pass a list of API values separated by comma using the following flag `--downloadSpecificAPI`.
+Instead of downloading all the configurations for all the API's you can pass a list of API values separated by comma using the following flag `--specific-api`.
 
 ```shell title="shell"
 
-$ monaco download --downloadSpecificAPI alerting-profiles,dashboard --environments=my-environment.yaml
+$ monaco download --specific-api alerting-profiles,dashboard --environments=my-environment.yaml
```

Comment:
could you please incorporate this doc fix in this change? `alerting-profile` rather than -profiles
https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/pull/445/files#diff-79e254c0d23f2ebc0c27f8151c7c71c52e226a9ca45a77134a9a58870d7dd347R27